### PR TITLE
Solana cNFT transfers: Use tree canopy option to reduce transaction size

### PR DESCRIPTION
**Pull request by bryan-eastwood3**
We had some issues where cNFT transfer transactions ended up being over the Solana transaction size limit. [This page](https://developers.metaplex.com/bubblegum/faq#replace-leaf-instruction-arguments) recommends using this flag to make them smaller. 

Tested by transferring an NFT with our custodial wallets API

### CODE REVIEW

File: typescript/packages/plugins/solana-nfts/package.json [hunk 1]
```diff
@@ -14,11 +14,13 @@
     "dependencies": {
         "@goat-sdk/core": "workspace:*",
         "@metaplex-foundation/digital-asset-standard-api": "1.0.0",
-        "@metaplex-foundation/mpl-bubblegum": "3.1.0",
+        "@metaplex-foundation/mpl-bubblegum": "4.2.1",
         "@metaplex-foundation/umi": "^0.9.2",
         "@metaplex-foundation/umi-bundle-defaults": "0.9.2",
         "@metaplex-foundation/umi-web3js-adapters": "0.8.10",
         "@solana/web3.js": "catalog:",
+        "got": "5.6.0",
```
Comment (devlyn37) [inline typescript/packages/plugins/solana-nfts/package.json:22]: intentional?

File: typescript/packages/plugins/solana-nfts/package.json [hunk 2]
```diff
@@ -14,11 +14,13 @@
...
+        "viem": "catalog:",
```
Comment (devlyn37) [inline typescript/packages/plugins/solana-nfts/package.json:23]: intentional?

**Review by devlyn37 — APPROVED** [review]
LGTM

File: typescript/packages/plugins/solana-nfts/package.json (hunk 1 above)
Comment (bryan-eastwood3) [reply typescript/packages/plugins/solana-nfts/package.json:22]: Don't get me started... without manually installing this dependency it tries to install a version that requires node <7 and fails... cost me 4 hours a few days ago

File: typescript/packages/plugins/solana-nfts/package.json (hunk 2 above)
Comment (bryan-eastwood3) [reply typescript/packages/plugins/solana-nfts/package.json:23]: No, I'll try removing

**Review by mPaella — APPROVED** [review]

**Pull request merged**